src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -618,18 +689,26 @@ public void testSetClasspath() {
         antTask.setClasspath(new Path(project, path2));
 
         final Path classpath = Whitebox.getInternalState(antTask, "classpath");
-        assertNotNull(classpath, "Classpath should not be null");
-        assertTrue(classpath.toString().contains(path1), "Classpath contain provided path");
-        assertTrue(classpath.toString().contains(path2), "Classpath contain provided path");
+        final String classpathString = classpath.toString();
+        assertWithMessage("Classpath should not be null")
+                .that(classpath)
+                .isNotNull();
+        assertWithMessage("Classpath does not contain provided path")
+                .that(classpathString)
+                .contains(path1);
+        assertWithMessage("Classpath does not contain provided path")
```

Comment:
Also can be one assert

Reply:
>
```
        assertWithMessage(errorMessage)
                .that(output.get(1))
                .startsWith("[WARN]")
                .contains("InputCheckstyleAntTaskError.java:4: ");
```
It gives compilation error. Are you sure this is valid?

Reply:
Sorry, by bad. I got confused with another framework.